src/promptflow/promptflow/_sdk/_service/app.py

```diff
@@ -39,10 +39,12 @@ def create_app():
         log_file = HOME_PROMPT_FLOW_DIR / PF_SERVICE_LOG_FILE
         log_file.touch(mode=read_write_by_user(), exist_ok=True)
         handler = logging.FileHandler(filename=log_file)
```

Comment:
we should limit the max size of our log file, maybe to 1MB

Reply:
limited the log file. It will rollover the log file when file size is nearly by 1M.
https://docs.python.org/3/library/logging.handlers.html#logging.handlers.RotatingFileHandler